comment-updater-core/src/main/kotlin/org/jetbrains/research/commentupdater/models/ONNXTensorUtils.kt

```diff
@@ -3,6 +3,10 @@ package org.jetbrains.research.commentupdater.models
 import ai.onnxruntime.OnnxTensor
 import ai.onnxruntime.OrtEnvironment
 
+/**
+ * Converts List arrays to ONNXTensors
```

Comment:
`list arrays`? judging by the code, it works with a list of lists or even lists of lists of lists